addons/survey/models/survey_user_input.py

```diff
@@ -591,10 +633,35 @@ class SurveyUserInputLine(models.Model):
     value_text_box = fields.Text('Free Text answer')
     suggested_answer_id = fields.Many2one('survey.question.answer', string="Suggested answer")
     matrix_row_id = fields.Many2one('survey.question.answer', string="Row answer")
+    answer_display_name = fields.Char('Answer', compute="_compute_answer_display_name",
```

Comment:
Why not overriding ``_compute_display_name`` and use display_name in view, as this is quite standard ?